skunkworks_crow/src/main/java/org/odk/share/activities/MainActivity.java

```diff
@@ -84,21 +86,17 @@ protected void onCreate(Bundle savedInstanceState) {
         sendForms.setEnabled(false);
 
         LinearLayoutManager llm = new LinearLayoutManager(this);
-        llm.setOrientation(LinearLayoutManager.VERTICAL);
+        llm.setOrientation(RecyclerView.VERTICAL);
         recyclerView.setLayoutManager(llm);
-        setupAdapter();
-        getSupportLoaderManager().initLoader(FORM_LOADER, null, this);
-        addListItemDivider();
-    }
 
-    @Override
-    protected void onResume() {
-        super.onResume();
-        if (isCollectInstalled()) {
-            updateAdapter();
-        } else {
-            showAlertDialog();
-        }
+        //check the storage permission and start the loader
+        setUpLoader();
+
+        emptyView.setOnClickListener((View view) -> {
```

Comment:
This should be removed as it is not intuitive and does not follow the general design of requesting permissions.

Reply:
removed the click listener on the empty view